lib/bot.js

```diff
@@ -170,15 +170,22 @@ class Bot {
   sendToDiscord(author, channel, text) {
     const discordChannelName = this.invertedMapping[channel.toLowerCase()];
     if (discordChannelName) {
-      // #channel -> channel before retrieving:
-      const discordChannel = this.discord.channels.find('name', discordChannelName.slice(1));
+      // #channel -> channel before retrieving and select only text channels:
+      const discordChannel = this.discord.channels.find(
+        (o) => o.name === discordChannelName.slice(1) && o.type === 'text'
+      );
 
       if (!discordChannel) {
         logger.info('Tried to send a message to a channel the bot isn\'t in: ',
           discordChannelName);
         return;
       }
 
+      if (typeof discordChannel.sendMessage !== 'function') {
```

Comment:
This will never happen now though right? Since you've filtered out the channels that aren't text.